.github/workflows/android.yml

```diff
@@ -52,9 +49,8 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Ruby
-        uses: ruby/setup-ruby@v1.118.0
+        uses: ruby/setup-ruby@v1
         with: 
-          ruby-version: ${{ inputs.ruby-version }}
```

Comment:
this is not needed as we have a .ruby-version file

Reply:
That is not how it works sadly!
The default ruby installer in github is not aware of the .ruby-version file.

Reply:
According to the docs it is

Reply:
nvm!